smart-contracts/wasm-transform/src/machine.rs

```diff
@@ -152,6 +161,7 @@ pub struct RuntimeStack {
 }
 
 #[derive(Debug)]
+/// A runtime error that we impose on top of the Wasm spec.
 pub enum RuntimeError {
     DirectlyCallImport,
```

Comment:
Could use a clarifying comment.
```suggestion
    /// Calling an imported function directly is not supported.
    DirectlyCallImport,
```